plugins/woocommerce/tests/php/src/Database/Migrations/CustomOrderTable/PostsToOrdersMigrationControllerTest.php

```diff
@@ -311,6 +311,10 @@ private function assert_order_op_data_is_migrated( $order ) {
 		$this->assertEquals( (float) $order->get_shipping_total(), (float) $db_order_op_data->shipping_total_amount );
 		$this->assertEquals( (float) $order->get_discount_tax(), (float) $db_order_op_data->discount_tax_amount );
 		$this->assertEquals( (float) $order->get_discount_total(), (float) $db_order_op_data->discount_total_amount );
+		$this->assertEquals(
+			$order->get_data_store()->get_download_permissions_granted( $order ),
```

Comment:
Seems like this data store method needs to be changed?

Reply:
🤦. You're absolutely right.